library/src/main/java/com/pixite/android/billingx/DebugBillingClient.kt

```diff
@@ -153,8 +148,9 @@ class DebugBillingClient(
 
   override fun launchBillingFlow(activity: Activity, params: BillingFlowParams): BillingResult {
     val intent = Intent(activity, DebugBillingActivity::class.java)
-    intent.putExtra(DebugBillingActivity.REQUEST_SKU_TYPE, params.skuType)
-    intent.putExtra(DebugBillingActivity.REQUEST_SKU, params.sku)
+    val skuDetails: List<SkuDetails> = params.zzj()
```

Comment:
Is there a non-obfuscated param available to get this detail?  This value could easily change if the source BillingFlowParams file is changed in any way.

Reply:
@rharter 
No. Unfortunately, only vrPurchaseFlow was not obfuscated in BillingFlowParams. This poor workaround is required, I agreed to your concern though.

Reply:
How about the following code?
```
    val skuDetails: List<SkuDetails> = params::class.java.declaredMethods
      .find {
        it.genericReturnType.toString() == "java.util.ArrayList<com.android.billingclient.api.SkuDetails>"
      }?.let { it.invoke(params) as ArrayList<SkuDetails> }
      .orEmpty()
```

Reply:
I've filed an issue [here](https://issuetracker.google.com/issues/191995386), as obfuscating everything there seems like an oversight and a poor choice.

I think the reflection approach is the only viable one at this point, as any even point release will probably change the method name.

Reply:
Thanks for filed the issue. I agree with you, so I've starred the issue.
I will use the reflection approach at this time. And if the Play Billing Library is fixed, I’ll fix the BillingX as well. Thank you.